markdown/org/docs/sewing/knit-binding/en.md

```diff
@@ -79,41 +80,50 @@ Stretch your binding too much, and it will spring back too far, causing the main
 
-This idea is that you cut your binding, join its edges, and then sew it in the opening. That is (in my opinion) no good.
+This idea is that you would cut your binding, join its edges to form a closed loop, and then sew it to the opening. However, this is not the best method.
 
-An alternative approach to sidestep the length issue is to attach the binding before you close the opening your binding goes in.
-For example, on a T-shirt you close one shoulder seam, attach the binding, then close the second shoulder seam.
+A possible alternate approach to sidestep the length issue would be to attach the binding before you close the opening your binding goes in.
+For example, on a T-shirt you would close one shoulder seam, attach the binding, then close the second shoulder seam.
 
-Obviously, that means your shoulder seam will cut through your binding which is a big no no in my book.
+However, that obviously means your shoulder seam would cut through your binding.
+So, this approach is also not desirable.
 
 ### Instead, do this
 
-- Cut your binding as long as the seam it needs to be sewn into. Not 90%, but 100%. This way, you know it's going to be too long, and that's what we want.
-- Mark the place you want your binding to be joined (say center back of the neckline) and place your binding 1cm beyond this point.
-- Start sewing your binding about 3cm or so from this point, and go all the way around. While you do so, make sure to adjust the stretch to whatever the curve requires. There's no tricks for this, you'll have to practice until it comes naturally to you.
-- When you make your way around and get close to the point you marked, stop about 3cm before that.
-- Now your entire binding is attached, apart from a 6cm or so stretch at the back. Pin both edges of your binding good sides together, making is exactly as long as needed and sew them together.
-- Sew the remaining 6cm of your binding.
+1. Cut your binding exactly as long as the seam it needs to be sewn into. Not 90%, but 100%. This way, you know it's going to be too long, and that's what we want.
+2. Mark the place you want your binding to be joined (say center back of the neckline) and place your binding 1 cm beyond this point.
+3. Start sewing your binding about 3 cm or so from this point, and go all the way around. While you do so, make sure to adjust the stretch to whatever the curve requires. There are no tricks for this; you'll have to practice until it comes naturally to you.
+4. When you make your way around and get close to the point you marked, stop about 3 cm before that. Now your entire binding is attached, apart from a 6 cm or so stretch at the back.
+5. Pin both edges of your binding good sides together to form the binding into a closed loop, making it exactly as long as needed.
+6. Cut off the excess binding, and sew the binding loop closed.
```

Comment:
This part I think is the relevant part that we needed to clarify people's problems with the documentation.